lib/data/list_account.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import '../constants/SourceString.dart';
+
+class Account {
+  static List<ListTile> listAccount = [
+    ListTile(
+      leading: Transform.scale(
+          scale: 1.5, child: Image.asset("assets/images/avatar.png")),
```

Comment:
Move path file image to SourceImage (like SourceString)

Reply:
i changed